crispycuts/src/reportWebVitals.ts

```diff
@@ -0,0 +1,15 @@
+import { ReportHandler } from 'web-vitals';
```

Comment:
Delete this file since i think it was only used in `index.tsx` and is a un-needed file

Reply:
Do a project scan to make sure nowhere else uses `reportWebVitals` though

Reply:
deleted the file, still need to do a project scan